moveit_core/trajectory_processing/test/test_time_optimal_trajectory_generation.cpp

```diff
@@ -410,6 +411,58 @@ TEST(time_optimal_trajectory_generation, testSingleDofDiscontinuity)
   }
 }
 
+TEST(time_optimal_trajectory_generation, testMimicJoint)
+{
+  const std::string urdf = R"(<?xml version="1.0" ?>
+      <robot name="one_robot">
+      <link name="base_link"/>
+      <link name="link_a"/>
+      <link name="link_b"/>
+      <joint name="joint_a" type="continuous">
+        <axis xyz="0 1 0"/>
+        <parent link="base_link"/>
+        <child link="link_a"/>
+        <limit effort="3" velocity="3"/>
+      </joint>
+      <joint name="joint_b" type="continuous">
+        <axis xyz="1 0 0"/>
+        <parent link="link_a"/>
+        <child link="link_b"/>
+        <mimic joint="joint_a" multiplier="2" />
+        <limit effort="3" velocity="3"/>
+      </joint>
+      </robot>)";
+
+  const std::string srdf = R"(<?xml version="1.0" ?>
+      <robot name="one_robot">
+        <virtual_joint name="base_joint" child_link="base_link" parent_frame="odom_combined" type="planar"/>
+        <group name="group">
+          <joint name="joint_a"/>
+          <joint name="joint_b"/>
+        </group>
+      "</robot>)";
```

Comment:
Looks like an extra quote here?

Reply:
Well spotted. Wasn't noticed, because the extra quote goes as text in the XML and is simply ignored.